gpii/node_modules/matchMakerFramework/test/data/app_specific_only.json

```diff
@@ -19,19 +19,19 @@
                     }
                 },
                 "http://registry.gpii.net/applications/com.microsoft.windows.cursors": {
-                    "No": "%SystemRoot%\\cursors\\aero_unavail_xl.cur",
-                    "Hand": "%SystemRoot%\\cursors\\aero_link_xl.cur",
-                    "Help": "%SystemRoot%\\cursors\\aero_helpsel_xl.cur",
-                    "Wait": "%SystemRoot%\\cursors\\aero_busy_xl.ani",
-                    "Arrow": "%SystemRoot%\\cursors\\aero_arrow_xl.cur",
-                    "NWPen": "%SystemRoot%\\cursors\\aero_pen_xl.cur",
-                    "SizeNS": "%SystemRoot%\\cursors\\aero_ns_xl.cur",
-                    "SizeWE": "%SystemRoot%\\cursors\\aero_ew_xl.cur",
-                    "SizeAll": "%SystemRoot%\\cursors\\aero_move_xl.cur",
-                    "UpArrow": "%SystemRoot%\\cursors\\aero_up_xl.cur",
-                    "SizeNESW": "%SystemRoot%\\cursors\\aero_nesw_xl.cur",
-                    "SizeNWSE": "%SystemRoot%\\cursors\\aero_nwse_xl.cur",
-                    "AppStarting": "%SystemRoot%\\cursors\\aero_working_xl.ani"
+                    "No": "%SystemRoot%\\cursors\\no_l.cur",
+                    "Hand": "",
```

Comment:
Why did all the cursors get changed from the aero versions, and why is "Hand" now missing?